client/v3/concurrency/session.go

```diff
@@ -19,10 +19,13 @@ import (
 	"time"
 
 	v3 "go.etcd.io/etcd/client/v3"
+	"go.uber.org/zap"
 )
 
 const defaultSessionTTL = 60
 
+var logger *zap.Logger
```

Comment:
Please no global logger

Reply:
+1

Reply:
@serathius @ahrtr, if you have the time and patience. Would you mind explaining the reason behind this? I am new to coding in golang. Or please drop some guides/blogs for my self-study.

Thanks.